### ISSUE TITLE
Sublime seems to ignore user settings

### ISSUE THREAD

**Issue by borkosavic** [drinks/sublime-initializr#2]
I set up custom url and new project path, but none are considered by sublime.

**Comment by drinks** [comment]
Are you on Windows, by chance? #1 has a fix for this but it's a breaking
change. I'll try to get it published out soon.

**Comment by borkosavic** [comment]
Nope, I'm on OSX 10.10.4 (14E46), using Sublime 3 (build 3074). If you need anything else, feel free to ask :)

**Comment by drinks** [comment]
If I could get a gist of your settings (even if obfuscated) that would be great.

**Comment by borkosavic** [comment]
Initializr.sublime-settings:
`{
    "zip_url": "http://www.initializr.com/builder?h5bp-content&html5shiv&h5bp-iecond&h5bp-chromeframe&h5bp-analytics&h5bp-htaccess&h5bp-favicon&h5bp-appletouchicons&h5bp-scripts&h5bp-robots&h5bp-humans&h5bp-404&h5bp-adobecrossdomain&h5bp-css&h5bp-csshelpers&h5bp-mediaqueryprint&h5bp-mediaqueries",
    "default_path": "~/Work/"
}
`
Preferences.sublime-settings:
`{
    "class": "tabset_control",
    "color_scheme": "Packages/Theme - itg.flat/itg.dark.tmTheme",
    "font_size": 14,
    "ignored_packages":
    [
        "Vintage"
    ],
    "theme": "itg.flat.dark.sublime-theme"
}
`

Is that what you asked for?
